v2/go.mod

```diff
@@ -5,7 +5,7 @@ go 1.18
 require (
 	github.com/google/go-cmp v0.5.8
 	google.golang.org/api v0.90.0
-	google.golang.org/genproto b98a9ff5e252
+	google.golang.org/genproto v0.0.0-20220722212130-b98a9ff5e252
```

Comment:
Do we need this change? Usually we prefer renovatebot to handle this I think.

Reply:
reverted the changes.  My local linter wants to fix it up, but doesn't seem to cause issues.